packages/shared/src/components/fields/TextField.tsx

```diff
@@ -137,6 +137,7 @@ export function TextField({
           },
           styles.field,
         )}
+        style={{ ...style, borderRadius: 'rounded-10' }}
```

Comment:
Oh sorry, actually meant we can apply if to the parent element. no need to set a style for it.

Reply:
This component already has rounded-10 (line 133: `compact ? 'h-9 rounded-10`'). Should I remove this line `style={{ ...style, borderRadius: 'rounded-10' }}`?

Reply:
Ah yes, then it can actually be removed as it would add them double.
But maybe double-check this after you remove it.